patches/server/0108-Prevent-chest-visual-open-bug.patch

```diff
@@ -0,0 +1,54 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Rejomy <sprytex@users.noreply.github.com>
+Date: Fri, 10 Jan 2025 21:02:29 +0500
+Subject: [PATCH] Prevent chest visual open bug.
+
+
+diff --git a/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java b/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
+index 1d8e4d1050ebafef8784920481621965fd5c212f..66bb8b59e406a54e10a2291e0615c3e2b6b24285 100644
+--- a/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
++++ b/src/main/java/com/hpfxd/pandaspigot/config/PandaSpigotConfig.java
+@@ -94,6 +94,9 @@ public class PandaSpigotConfig {
+     public static PandaSpigotConfig get() {
+         return config;
+     }
++
++    @Comment("Prevent chest and sound bugs.")
++    public boolean dropUsePacketsInInventory = true;
+     
+     //------------------------------------------------------------------------
+     @Comment("On servers with plugins that constantly churn through scoreboards, there is a risk of\n" +
+diff --git a/src/main/java/net/minecraft/server/PlayerConnection.java b/src/main/java/net/minecraft/server/PlayerConnection.java
+index 40dc1eb6a86dede8377775ad2776fd9193d24cbc..0b1d3f5c79bf3ca9a9b2a3b51f39db225493d8c5 100644
+--- a/src/main/java/net/minecraft/server/PlayerConnection.java
++++ b/src/main/java/net/minecraft/server/PlayerConnection.java
+@@ -4,6 +4,7 @@ import com.google.common.collect.Lists;
+ import com.google.common.primitives.Doubles;
+ import com.google.common.primitives.Floats;
+ import com.google.common.util.concurrent.Futures;
++import com.hpfxd.pandaspigot.config.PandaSpigotConfig;
```

Comment:
I don't know how to say this in a simple way, but instead of importing it is preferable to use com.hpfxd.pandaspigot.config.PandaSpigotConfig.get() directly below, to avoid unnecessary diff